recipes/vtk/all/conanfile.py

```diff
@@ -0,0 +1,139 @@
+from conans import ConanFile, CMake, tools
+import os
+import json
+
+
+class VtkConan(ConanFile):
+    name = "vtk"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://vtk.org/"
+    description = "VTK is an open-source software system for" \
+        "image processing, 3D graphics, volume rendering and visualization."
+    topics = ("conan", "vtk", "geometry", "algorithms")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False],
+               "build_testing": ["ON", "OFF", "WANT"],
+               "vtk_smp_implementation": ["Sequential", "OpenMP", "TBB", "STDThread"],
+               "vtk_wrap_python": [True, False],
+               "vtk_use_cuda": [True, False],
+               "vtk_use_mpi": [True, False],
+               "vtk_group_enable_qt": ["YES", "WANT", "DONT_WANT", "NO", "DEFAULT"],
+               "vtk_group_enable_rendering": ["YES", "WANT", "DONT_WANT", "NO", "DEFAULT"],
+               "vtk_group_enable_mpi": ["YES", "WANT", "DONT_WANT", "NO", "DEFAULT"],
+               "vtk_group_enable_standalone": ["YES", "WANT", "DONT_WANT", "NO", "DEFAULT"],
+               "vtk_group_enable_views": ["YES", "WANT", "DONT_WANT", "NO", "DEFAULT"],
+               "vtk_group_enable_web": ["YES", "WANT", "DONT_WANT", "NO", "DEFAULT"],
+               "vtk_group_enable_imaging": ["YES", "WANT", "DONT_WANT", "NO", "DEFAULT"],
+               "fPIC": [True, False]}
+    default_options = {"shared": False,
+                       "build_testing": "OFF",
+                       "vtk_smp_implementation": "Sequential",
+                       "vtk_wrap_python": False,
+                       "vtk_use_cuda": False,
+                       "vtk_use_mpi": False,
+                       "vtk_group_enable_imaging": "DEFAULT",
+                       "vtk_group_enable_qt": "DEFAULT",
```

Comment:
Our convention is `with_<dep>` 

https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#recommended-feature-options-names